AEPAudience/Sources/URL+Audience.swift

```diff
@@ -19,7 +19,8 @@ extension URL {
     /// Creates a new Audience Manager hit URL
     /// - Parameters:
     ///   - state: the Audience State containing Audience-related variables.
-    static func buildAudienceHitURL(state: AudienceState?) -> URL? {
+    ///   - data: new signal data to be sent
+    static func buildAudienceHitURL(state: AudienceState?, data: [String: String]) -> URL? {
```

Comment:
should we have default value` data: [String: String] = [:]`

Reply:
I think data should always be passed in when building the hit, even if that may be empty

Reply:
I think data should always be passed in when building the hit, even if that may be empty